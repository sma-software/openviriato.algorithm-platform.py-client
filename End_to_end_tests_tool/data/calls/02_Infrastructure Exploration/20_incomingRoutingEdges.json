{
    "restResource": "/nodes/{id}/incoming-routing-edges",
    "httpMethod": "GET",
    "urlParameters": [ "161" ],
    "queryParameters": [ ],
    "requestBody": null,
    "CSharpMethodName": "GetIncomingRoutingEdges(routingPoint)",
    "CSharpMethodParameterTypes": [ "SMA.AlgorithmPlatform.AlgorithmInterface.AIDM.RoutingPoint" ],
    "subcategory": "Get Incoming Routing Edges"
}