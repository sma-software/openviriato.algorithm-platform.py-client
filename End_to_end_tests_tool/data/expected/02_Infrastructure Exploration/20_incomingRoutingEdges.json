{
  "RoutingEdges":
  [
    {
      "NodeID": 161,
      "EndNodeTrackID": 162,
      "StartSectionTrackID": 885
    },
    {
      "NodeID": 161,
      "EndNodeTrackID": 163,
      "StartSectionTrackID": 885
    },
    {
      "NodeID": 161,
      "EndNodeTrackID": 162,
      "StartSectionTrackID": 886
    },
    {
      "NodeID": 161,
      "EndNodeTrackID": 163,
      "StartSectionTrackID": 886
    },
    {
      "NodeID": 161,
      "EndNodeTrackID": 162,
      "StartSectionTrackID": 887
    },
    {
      "NodeID": 161,
      "EndNodeTrackID": 163,
      "StartSectionTrackID": 887
    }
  ]
}
